R/add-tbl.R

```diff
@@ -83,3 +78,18 @@ check_new_tbls <- function(dm, tbls) {
   }
 }
 
+dm_try_tables <- function(quo, table_names) {
```

Comment:
```suggestion
dm_try_tables <- function(code, table_names) {
```

and later

```r
quo <- enquo(code)
```

But we don't need the `quo` later. Deferred evaluation works with plain R, see https://github.com/tidyverse/tibble/blob/master/R/msg.R#L297 for a similar implementation. Can you please model after this?